civihr_employee_portal/civihr_employee_portal.module

```diff
@@ -5486,6 +5488,50 @@ function civihr_employee_portal_form_views_exposed_form_alter(&$form, &$form_sta
             }
         }
     }
+
+    if (substr($form['#id'], 0, 33) === 'views-exposed-form-civihr-report-') {
```

Comment:
Because there isn't a specific `$form['#id']` value.
At http://localhost:8900/reports/people the value is `views-exposed-form-civihr-report-people-default`.
At http://localhost:8900/reports/leave_and_absence/ the value is `views-exposed-form-civihr-report-leave-and-absence-default`.